packages/open-payments/src/client/requests.ts

```diff
@@ -128,6 +129,12 @@ export const createAxiosInstance = (args: {
   if (args.privateKey && args.keyId) {
     axiosInstance.interceptors.request.use(
       async (config) => {
+        if (config.data) {
+          config.headers['Content-Digest'] = createContentDigestHeader(
+            JSON.stringify(config.data),
+            ['sha-512']
+          )
```

Comment:
Upon further inspection, `Content-Length` and `Content-Type` headers aren't yet populated in the interceptor
```suggestion
          const data = JSON.stringify(config.data)
          config.headers['Content-Digest'] = createContentDigestHeader(data, [
            'sha-512'
          ])
          config.headers['Content-Length'] = Buffer.from(data, 'utf-8').length
```
You could either also add `Content-Type` here or above as
```
axiosInstance.defaults.headers.post['Content-Type'] = 'application/json'
```
but then I think we'd need to go back to https://github.com/interledger/rafiki/pull/722#discussion_r1016045236